.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 env:
   # Common versions
-  GO_VERSION: '1.22.3'
+  GO_VERSION: '1.22.8'
```

Comment:
```suggestion
  GO_VERSION: '1.21.13'
```

Should we keep this as 1.21 as well?

Reply:
Ah ok. Inconsistency between ci.yaml and go.mod was already in place and not introduced with this PR. 

Then, it sounds good to me 👍